robomimic/utils/file_utils.py

```diff
@@ -383,13 +355,19 @@ def download_url(url, download_dir):
     """
 
     # check if url is reachable. We need the sleep to make sure server doesn't reject subsequent requests
-    assert url_is_alive(url), "@download_url got unreachable url: {}".format(url)
+    response = requests.get(url, stream=True)
     time.sleep(0.5)
```

Comment:
The purpose of this time.sleep was to avoid the problem that you ran into (the 503 error) due to too many requests to the server (one for checking the url is valid, and one for downloading from the url), but it seems like the amount of time might not have been large enough for your setup. Changing to use the `requests` module might have fixed things for you because you only make 1 url request instead of 2 now.

Can you verify on your end that this is indeed the issue, by just commenting the assertion and the time.sleep out in the original code?